### ISSUE TITLE
Long line should be wrapped

### ISSUE THREAD

**Issue by hoangdat** [linagora/tmail-flutter#157]
### Current behavior

![long_line_should_be_wrapped.jpeg](https://images.zenhubusercontent.com/5f7d8ef759d36aecc684e2a4/dfa34087-4614-418f-854b-ca50d42c9982)

**Comment by hoangdat** [comment]
@dab246  
that line is a signature, which is wrapped in a `<pre>` tag. I still not find the way to send it to you

**Comment by hoangdat** [comment]
> maybe we could have an horizontal scroll bar on too long pre?
> 
> Like on github?
> 
> ```
> loooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooong pre
> ```
> 
> wrapping the line is also acceptable to me.

yes, maybe, we should put more tag to that html. That the first idea from me. We will have more discussion on that next days

**Comment by chibenwa** [comment]
This is no longer happening, correct?
